io.openems.edge.evcs.alfen/src/io/openems/edge/evcs/alfen/Alfen.java

```diff
@@ -0,0 +1,158 @@
+package io.openems.edge.evcs.alfen;
+
+import io.openems.common.channel.AccessMode;
+import io.openems.common.channel.Level;
+import io.openems.common.channel.PersistencePriority;
+import io.openems.common.channel.Unit;
+import io.openems.common.exceptions.OpenemsError.OpenemsNamedException;
+import io.openems.common.types.OpenemsType;
+import io.openems.edge.common.channel.Doc;
+import io.openems.edge.common.channel.WriteChannel;
+import io.openems.edge.common.component.OpenemsComponent;
+
+
+public interface Alfen extends OpenemsComponent {
+
+	public enum ChannelId implements io.openems.edge.common.channel.ChannelId {
```

Comment:
The EVCS interface has changed in the last couple of weeks. We need to check if this still compiles with the new interface